ydb/core/tx/schemeshard/schemeshard_utils.cpp

```diff
@@ -348,39 +351,47 @@ void SetImplTablePartitionConfig(
     *tableDescription.MutablePartitionConfig() = PartitionConfigForIndexes(baseTablePartitionConfig, indexTableDesc);
 }
 
- void FillIndexImplTableColumns(
-    const THashMap<ui32, struct NSchemeShard::TTableInfo::TColumn>& baseTableColumns,
-    const NTableIndex::TTableColumns& implTableColumns,
+void FillIndexImplTableColumns(
+    const auto& baseTableColumns,
+    const TTableColumns& implTableColumns,
     NKikimrSchemeOp::TTableDescription& implTableDesc)
 {
-    //Columns and KeyColumnNames order is really important
-    //the order of implTableColumns.Keys is the right one
+    const auto was = implTableDesc.ColumnsSize();
```

Comment:
Please double check, whether it is right. In current version there is no need in it.

Reply:
in current version sort is not really correct because same ids is sorted

Reply:
The difference is I sort only elements that was added in this function

Reply:
implTableDesc.ColumnsSize() is always 0.

Reply:
it's not true